### DISCUSSION TITLE
Where are the default configuration files?

### DISCUSSION THREAD

**Discussion by th3worst4** [NvChad/NvChad#3088] [Q&A] [answered]
May be a naive question, but I can't find the configuration files of the default plugins.
For example, the cmp-nvim configuration is pretty good but I would like to add the "calc" option to the buffer. If I add this option under the `nvim/lua/plugins/init.lua` all default configuration is overwritten. I want to add just a line to the default setup, but I first need to find it. So where is it?

**Comment by siduck** [comment, marked as answer]
override cmp plugin, go read plugins doc on nvchad.com
